rdflib/plugins/parsers/notation3.py

```diff
@@ -1745,7 +1746,7 @@ def newBlankNode(self, arg=None, uri=None, why=None):
             return arg.newBlankNode(uri)
         elif isinstance(arg, Graph) or arg is None:
             self.counter += 1
-            bn = BNode("n" + str(self.counter))
+            bn = BNode("n%sb%s" % (self.uuid, self.counter))
```

Comment:
I'm going to have to look into this a little deeper. I don't know if adding a UUID to the RDFSink is the best way to approach this.

Reply:
UUID usage is what is currently done internally by the formula blank node generator